tools/record.sh

```diff
@@ -113,11 +126,27 @@ done
       then
         (
           username=${id_username[$id]}
+          
           errf '[%s] stream started\a\n' "$username"
           while :
           do
             mkdir -p "$username"
-            filename=$(printf "%s/%s.ts" $username "$(date -u '+%Y_%m_%d_%H_%M_%S_(%Z)')")
+
+            filename="$username/"
+            
+            if [ "$FILENAME_TITLE" == "1" ];
+              then
+                title=$(get_stream_title $username)
+                filename+="${title}_"
+            fi
+            if [ "$FILENAME_DATE" == "1" ];
+              then
+                filename+=$(date $DFORMAT)
+              else
+                filename+=$(date -u '+%Y-%m-%d-%H-%M-%S-(%Z)')
```

Comment:
```suggestion
                filename+=$(date -u '+%Y_%m_%d_%H_%M_%S_(%Z)')
```

I would also prefer the date format stay the same as it was before, `+%Y_%m_%d_%H_%M_%S_(%Z)`